docs/Optimization-for-Nintendo-64.md

```diff
@@ -31,7 +31,7 @@ then cancel, exit and reboot.
 
 ## Selecting the Correct Emulator and Graphics Plugin
 
-Just as important as overclocking, selecting the right emulator/graphics plugin from the [runcommand](https://github.com/RetroPie/RetroPie-Setup/wiki/runcommand) menu on a per game basis will also increase performance. Selecting the right plugin can make all the difference in making a game playable. 
+Just as important as overclocking, selecting the right emulator/graphics plugin from the [runcommand](runcommand) menu on a per game basis will also increase performance. Selecting the right plugin can make all the difference in making a game playable. 
```

Comment:
`[runcommand](runcommand) menu` should be `[Runcommand launch menu](Runcommand##runcommand-launch-menu)`.